src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -102,3 +109,50 @@ def get_projects_and_filenames_from_source_file(
                 pr_filename.replace(code_mapping.source_root, code_mapping.stack_root)
             )
     return project_list, sentry_filenames
+
+
+def get_top_5_issues_by_count_for_file(
+    projects: List[Project], sentry_filenames: List[str]
+) -> list[dict[str, Any]]:
+    """Given a list of issue group ids, return a sublist of the top 5 ordered by event count"""
+    group_ids = list(
+        Group.objects.filter(
+            last_seen__gte=datetime.now() - timedelta(days=14),
+            project__in=projects,
+        ).values_list("id", flat=True)
+    )
+    project_ids = [p.id for p in projects]
+
+    request = SnubaRequest(
+        dataset=Dataset.Events.value,
+        app_id="default",
+        tenant_ids={"organization_id": projects[0].organization_id},
+        query=(
+            Query(Entity("events"))
+            .set_select(
+                [
+                    Column("group_id"),
+                    Function("count", [], "event_count"),
+                    Function("uniq", [Column("user_hash")], "affected_users"),
+                    Function("isHandled", [], "is_handled"),
+                ]
+            )
+            .set_groupby([Column("group_id"), Column("exception_stacks.mechanism_handled")])
+            .set_where(
+                [
+                    Condition(Column("project_id"), Op.IN, project_ids),
+                    Condition(Column("group_id"), Op.IN, group_ids),
+                    Condition(Column("timestamp"), Op.GTE, datetime.now() - timedelta(days=30)),
```

Comment:
Do we need to check for the past 30 days of events if we are searching issues from the past 14 days?

Reply:
is there a difference between `timestamp` and `last_seen`?

Reply:
but yes it seems like it should be just 14

Reply:
The event `timestamp` is the time at which the event was ingested. The `last_seen` is the timestamp of the most recent event in the issue.

Reply:
changed to check for timestamp >= 14 days ago